xarray/core/_typed_ops.pyi

```diff
@@ -0,0 +1,731 @@
+"""Stub file for mixin classes with arithmetic operators."""
+# This file was generated using xarray.util.generate_ops. Do not edit manually.
+
+from typing import NoReturn, TypeVar, Union, overload
+
+import numpy as np
+
+from .dataarray import DataArray
+from .dataset import Dataset
+from .groupby import DataArrayGroupBy, DatasetGroupBy, GroupBy
+from .variable import Variable
+
+try:
+    from dask.array import Array as DaskArray
+except ImportError:
+    DaskArray = np.ndarray
+
+T_Dataset = TypeVar("T_Dataset", bound="DatasetOpsMixin")
```

Comment:
Maybe add a comment that `DatasetOpsMixin` etc are parent classes of `Dataset`.

Reply:
:+1: Done.